database/migrations/20230613204013-initial-schema.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
```

Comment:
I made some lower-level DB comments but if it's easy to reset/modify the schema later then feel free to ignore the nitpicks.